components/table/table.module.css

```diff
@@ -0,0 +1,5 @@
+.table {
+  background-color: var(--table-color);
+  color: var(--primary-font-color);
+  box-shadow: 0 0.2rem 0.4rem 0 #00000029;
```

Comment:
Just a quick idea - if you are going to use this box-shadow more often, you could also store it in a global variable

Reply:
good idea! thanks!